### PR TITLE
Sandrine's dog rescue

### PR DESCRIPTION
**Pull request by MindstormingAB**
I first created this pull request in desperation, when I was stuck with flexbox, hoping that somebody will revise my code and give me the solution. But I solved the issue the day after and closed the pull request! :-)  Now, I feel satisfied with what I accomplished. See my next comment for a description of this project.

### CODE REVIEW

**Comment by MindstormingAB** [comment]
I just found the solution!

**Comment by MindstormingAB** [comment]
For this news site project, I chose to build a webpage about a dog rescue center, using the news section as a picture gallery for dogs in need of an adoption.
I applied the "mobile first" approach, using different media queries for tablet and desktop versions.
I used the flexbox technique in the big news and other news sections and the CSS-grid technique in the header.
I added a hover effect to the images on the cards, a box shadow to all images and a "beating heart" animation.
I have tested my web page with good results in Chrome, Edge, on iPhone XR and on iPad Pro but it does not look good at all in Internet Explorer; I tried to solve that issue by adding alternative CSS codes but it almost broke my initial solution so I went back to my previous code. What could I have done to make it compatible with Internet Explorer?

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,338 @@
...
+  margin: 30px 30px 10px 30px;
+}
+
+.cards a figcaption {
+  margin: -5px;
+}
+
+.description {
+  display: none;
+}
```
Comment (HIPPIEKICK) [inline code/style.css:188]: Remove unused code :)

**Review by HIPPIEKICK — APPROVED** [review]
Great job with the responsiveness, and super that you tried out CSS grid! And what a cute animation! Wow, you’ve done a lot of testing which is great. Regarding Internet Explorer - I wouldn’t stress to much about that as Internet Explorer will soon be out of use: https://www.independent.co.uk/life-style/gadgets-and-tech/news/microsoft-internet-explorer-out-use-11-edge-a9676176.html

In week 8 we will talk more about cross browser testing, but if you’re curious already - check out this page: https://caniuse.com/#home or google on browser compatibility. For now - I think you’ve done enough testing on the devices and browsers you mentioned.

Keep up the good work Sandrine!

**Pull request closed without merging**